src/Features/VisualBasic/Portable/CodeFixes/OverloadBase/OverloadBaseCodeFixProvider.vb

```diff
@@ -31,6 +31,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.CodeFixes.OverloadBase
             End If
 
             context.RegisterCodeFix(New AddOverloadsKeywordAction(context.Document, token.Parent), context.Diagnostics)
+            context.RegisterCodeFix(New AddShadowsKeywordAction(context.Document, token.Parent), context.Diagnostics)
```

Comment:
❗️ We don't want both code fixes available for both warnings. This should do one of the following instead:

1. Register the single correct code fix based on the actual diagnostic ID of `diagnostic`
2. Create a single code action (e.g. `AddMethodModifierAction`) which takes the modifier as a constructor argument, and use the actual diagnostic ID of `diagnostic` to determine which argument to pass to the constructor

Reply:
@sharwell while it would be out of scope for @06needhamt , consider just removing the code action entirely.  There does not seem to be any purpose for it.  A normal DocumentChangeCodeAction would suffice here just fine, and allow the actual interesting logic to stay here instead of being pushed into the CodeAction

Reply:
Fixed by registering the correct code fix based on the actual diagnostic ID of diagnostic